202312-quarto-reports/2023-12-quarto-reports.qmd

```diff
@@ -0,0 +1,278 @@
+| What is Quarto? What can it do?             | 11:05 - 11:15 | Creating a report from your script | 12:05 - 12:35 |
+| Creating Quarto reports from .R, .Rmd files | 11:15 - 11:45 | Sharing your report                | 12:35 - 12:45 |
+| Ways to share Quarto reports                | 11:45 - 11:55 | Where to get more help             | 12:45 - 12:50 |
+
+# Welcome!
+
+::: notes
+Slot here to show results of pre-workshop survey on who here knows about/uses Quarto.
+:::
+
+## Before we get started {.smaller}
+
+::: incremental
+-   These slides (and links!) are available at: <https://cct-datascience.quarto.pub/sharing-research-as-quarto-reports/>
+-   Please make sure you have RStudio (ver. \>= 2022.07.1) and Quarto installed.
+    -   You can check your RStudio version at "Help \> About RStudio". The most recent version of RStudio is available [here](https://posit.co/download/rstudio-desktop/).
+    -   You can check if Quarto is installed by opening a terminal and running `quarto --version`. Install Quarto [here](https://quarto.org/docs/get-started/).
+:::
+
+## Learning objectives
+
+::: incremental
+-   Know what Quarto is, what it can do, and how it differs from specific *programming languages* and *markup languages* you may know of.
+-   Create your own Quarto report documenting your research.
+-   Share your report using Quarto Pub, Posit Connect, email, and/or Github.
+-   Know where to go for additional help and/or adventures in Quarto.
+:::
+
+## What is Quarto?
+
+::: incremental
+-   *"An open-source scientific publishing system"*
+-   Integrate code, results, and formatted text within one document
+-   Document types range from static .html pages, to slideshows, to interactive dashboards!
+-   Published by Posit (formerly RStudio)
+:::
+
+## What is Quarto?
+
+![](quarto_order.png)
+
+::: notes
+"Open-source publishing system": Sounds vague, but this is what it is.
+
+More specifically, it's a framework for integrating code, executing that code and showing the results, and text into one document.
+
+Document types can be broad.
+
+Published by Posit, but it's *free* and *not just for R*.
+
+This general type of system is called "Literate programming".
+:::
+
+## A Quarto example
+
+-   [An example Quarto research report](https://cct-datascience.quarto.pub/quarto-workshop-example)
+
+    -   Or go to the URL: <https://cct-datascience.quarto.pub/quarto-workshop-example>
+
+-   [Source code for this example](https://github.com/diazrenata/squareone-quarto/blob/main/s3_biomass_as_quarto.qmd)
+
+    -   Or go to the URL: <https://github.com/diazrenata/squareone-quarto/blob/main/s3_biomass_as_quarto.qmd>
+
+::: notes
+Point out:
+
+-   Formatting
+-   In-line code, figures, tables
+-   Figure captions
+-   Citations
+-   Scholarly author tracking
+:::
+
+## Cool, but tricky. What are the benefits?
+
+::: incremental
+-   Reproducibility
+
+-   Dynamic updating
```

Comment:
You may already be planning on emphasizing this, but I think this can be a huge pro for people. Especially saving time and errors changing individual numbers in a manuscript, for example.

Reply:
Yes!